kratos/tests/test_mortar_mapper.py

```diff
@@ -0,0 +1,187 @@
+            #"sub_model_part_name"  : "Parts_Parts_Auto1"
+        #}
+        #""")
+
+        #out_parameters["output_file_name"].SetString(input_filename+".json")
+
+        #out = json_output_process.JsonOutputProcess(self.StructureModel, out_parameters)
+        #out.ExecuteInitialize()
+        #out.ExecuteBeforeSolutionLoop()
+        #out.ExecuteFinalizeSolutionStep()
+
+    def test_less_basic_mortar_mapping_triangle(self):
+        input_filename = os.path.dirname(os.path.realpath(__file__)) + "/mortar_mapper_python_tests/test_integration_several_triangles"
+        self._mapper_tests(input_filename, 3, False)
+
+    def test_simple_curvature_mortar_mapping_triangle(self):
+        input_filename = os.path.dirname(os.path.realpath(__file__)) + "/mortar_mapper_python_tests/test_simple_curvature"
+        self._mapper_tests(input_filename, 3, False)
+
+    def test_mortar_mapping_triangle(self):
+        input_filename = os.path.dirname(os.path.realpath(__file__)) + "/mortar_mapper_python_tests/test_double_curvature_integration_triangle"
+        self._mapper_tests(input_filename, 3, False)
+
+    def test_mortar_mapping_quad(self):
+        input_filename = os.path.dirname(os.path.realpath(__file__)) + "/mortar_mapper_python_tests/test_double_curvature_integration_quadrilateral"
+        self._mapper_tests(input_filename, 4, False)
+
+    def __post_process(self):
+        from gid_output_process import GiDOutputProcess
+        self.gid_output = GiDOutputProcess(self.main_model_part,
+                                    "gid_output",
+                                    KratosMultiphysics.Parameters("""
+                                        {
+                                            "result_file_configuration" : {
+                                                "gidpost_flags": {
+                                                    "GiDPostMode": "GiD_PostBinary",
+                                                    "WriteDeformedMeshFlag": "WriteUndeformed",
+                                                    "WriteConditionsFlag": "WriteElementsOnly",
+                                                    "MultiFileFlag": "SingleFile"
+                                                },
+                                                "nodal_results"       : ["DISPLACEMENT","NORMAL","TEMPERATURE"],
+                                                "nodal_nonhistorical_results": ["NODAL_AREA","NODAL_MAUX","NODAL_VAUX"]
+                                            }
+                                        }
+                                        """)
+                                    )
+
+        self.gid_output.ExecuteInitialize()
+        self.gid_output.ExecuteBeforeSolutionLoop()
+        self.gid_output.ExecuteInitializeSolutionStep()
+        self.gid_output.PrintOutput()
+        self.gid_output.ExecuteFinalizeSolutionStep()
+        self.gid_output.ExecuteFinalize()
+
+    def __sci_str(self, x):
```

Comment:
what does this function do again? 
Just curious

Reply:
It forces the scientific notation